Descent3/localization.cpp

```diff
@@ -218,69 +206,39 @@ int LoadStringTables(void) {
 }
```

Comment:
At this point, String_table_size should match String_table.size(), maybe some checks to actually assert that can't hurt either.